magicctapipe/io/io.py

```diff
@@ -30,7 +29,7 @@
 except ImportError:
     from importlib_resources import files
 
-from ..utils import NO_MAGIC_EVENTS, calculate_mean_direction, transform_altaz_to_radec
```

Comment:
something went wrong when updating the branch, because this (changes in lines 9, 33, 741 and 755)  would remove another change that was merged recently.